android/src/main/java/com/ahm/capacitor/camera/preview/CameraPreview.java

```diff
@@ -386,6 +387,23 @@ public void stopRecordVideo(PluginCall call) {
         // call.resolve();
     }
 
+    @PluginMethod(returnType = PluginMethod.RETURN_CALLBACK)
+    public void subscribeToFocusSet(PluginCall call) {
```

Comment:
Wie beim anderen PR angemerkt wäre es schön das Callback in den `start` Call reinzubringen und automatisch dann wieder beim `stop` zu entfernen

Reply:
Nachdem der Call selbst so zusagen der Callback ist, würde ich die subscription als eigenen Call belassen. Sollte irgendwann mehr als ein Callback gebrauch werden, müsste man das ohnehin so machen und das ist dann leicht erweiterbar.

Ist jetzt einfach ein Vorschlag. Es geht schon anders auch aber ich würde es nicht so machen